### DISCUSSION TITLE
SvelteKit Tailwind installation is outdated

### DISCUSSION THREAD

**Discussion by chadrogersmudbath** [tailwindlabs/tailwindcss#12011] [Help]
I just followed the svelteKit and tailwind docs https://tailwindcss.com/docs/guides/sveltekit and my VS Code editor does not recognize @tailwind and colors in "theme(colors.gray.100)".

I followed the docs consistently and installed the Tailwind Css IntelliSense extension.

How do I make these errors go away?

![image](https://github.com/tailwindlabs/tailwindcss/assets/144183087/7d295a00-2c9b-424a-9838-557ff29ee14a)
![image](https://github.com/tailwindlabs/tailwindcss/assets/144183087/7c658de4-376f-4ccc-a79b-e30f2c8f95c1)

**Comment by RobinMalfait** [comment]
Hey!

The docs are complete in the sense that they tell you how to setup Tailwind CSS with SvelteKit. Once you get to the `npm run dev` step you will notice that it does work as expected.

The errors you are seeing are not errors but warnings and are specific to your editor setup.

For the first warning in your `app.css` file, change the language in vscode to `tailwindcss`. You can do this:
<img width="739" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/1834413/cc97a1c9-934a-4e1f-b491-d88f5eb75824">

Then:
<img width="747" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/1834413/28726bb9-5586-471c-9283-073552f77aa2">

For your second screenshot, make sure you are using the latest version of the "Svelte for VS Code" plugin and make sure the language is set to `svelte` in that file:

<img width="877" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/1834413/ef14d75f-0a17-4e32-8d5d-46479ab12dee">
